examples/databricks/log_runs.py

```diff
@@ -0,0 +1,45 @@
+"""
+Logs MLflow runs in Databricks.
```

Comment:
Can we clarify that this is used for logging to Databricks from an external host? We don't want users to think this is necessary for using MLflow on Databricks.

Reply:
Can we simply add `from an external host`?

```suggestion
Logs MLflow runs in Databricks from an external host.
```

Reply:
Sounds good!